### DISCUSSION TITLE
Auto-replace (--) with em dash

### DISCUSSION THREAD

**Discussion by savely-krasovsky** [outline/outline#6477] [Ideas]
Many writing apps and messengers replacing two consecutive hyphens with em or en dashes (usually em one). I was surprised that Outline does not seem to support it.

**Comment by tommoor** [comment]
Could do – the rules for auto-replace are defined here:
https://github.com/outline/outline/blob/main/app/editor/extensions/SmartText.ts

**Reply by savely-krasovsky** [reply]
Created a PR: https://github.com/outline/outline/pull/6479
